django/template/defaultfilters.py

```diff
@@ -422,15 +452,16 @@ def linebreaks_filter(value, autoescape=True):
 
 @register.filter(is_safe=True, needs_autoescape=True)
 @stringfilter
+@autoescape_aware
 def linebreaksbr(value, autoescape=True):
     """
     Convert all newlines in a piece of plain text to HTML line breaks
     (``<br>``).
     """
     autoescape = autoescape and not isinstance(value, SafeData)
     value = normalize_newlines(value)
-    if autoescape:
-        value = escape(value)
+    # if autoescape:
```

Comment:
Why is the code commented out?

Reply:
The code should be removed rather than commented. I added the autoescape_aware decorator to the function, which takes care of the escaping, so it no longer needs to escape the value in the function.

Reply:
I see. Just go ahead and remove it then. It's visible the diff anyways, and we tend to use comments mostly to explain code bits that are hard to read. You can add a comment, that it will be escaped by the decorator, but that's also kind of visible because of the decorator in the code.